yolapy/services.py

```diff
@@ -1,14 +1,24 @@
 from demands import HTTPServiceClient
+
 from yolapy.configuration import config as defaults
 from yolapy.resources import (
-    campaign, cname_zone, partner, site, siteimport, subscription, user)
+    campaign,
+    cname_zone,
+    partner,
+    site,
+    siteimport,
+    subscription,
+    user,
+    wl_task,
+)
 
 
 class Yola(
         HTTPServiceClient, campaign.CampaignResourceMixin,
         cname_zone.CnameZoneMixin, partner.PartnerResourceMixin,
         site.SiteResourceMixin, siteimport.SiteImportResourceMixin,
-        subscription.SubscriptionResourceMixin, user.UserResourceMixin):
+        subscription.SubscriptionResourceMixin, user.UserResourceMixin,
+        wl_task.WLTaskResourceMixin):
```

Comment:
What do you think about every mixin in separate line?

Reply:
> What do you think about every mixin in a separate line?

👍 I actually like such style more and more than happy to improve it!